src/libs/antares/series/series.cpp

```diff
@@ -31,9 +33,31 @@ using namespace Yuni;
 
 namespace Antares::Data
 {
+void TimeSeries::Numbers::registerSeries(const TimeSeries* s)
+{
+    series.push_back(s);
+}
+
+// TODO[FOM] Code duplication
+static bool checkAllElementsIdenticalOrOne(std::vector<uint> w)
+{
+    auto first_one = std::remove(w.begin(), w.end(), 1); // Reject all 1 to the end
```

Comment:
Not important but note that it's not the most efficient way of implementing this functions logic: without copying and moving anything, we could just register the first value different from 1, continue, and stop as soon as we encounter another one.

Reply:
Yes, we had issues in the past with this function but you're right. We have a lot of UTs already so we could change implementation